### PR TITLE
frontend/bitsurance: localize bitsurance terms privacy link

### PR DESCRIPTION
**Pull request by Beerosagos**
Bitsurance terms and conditions page contains an external link towards the bitsurance website. This commit introduces different links for german and non german languages.

### CODE REVIEW

File: frontends/web/src/components/terms/bitsurance-terms.tsx [hunk 1]
```diff
@@ -30,6 +33,14 @@ export const BitsuranceTerms = ({ onAgreedTerms }: TProps) => {
   const handleSkipDisclaimer = (e: ChangeEvent<HTMLInputElement>) => {
     setConfig({ frontend: { skipBitsuranceDisclaimer: e.target.checked } });
   };
+  const getPrivacyLink = (): string => {
+    const selectedLanguage = defaultLanguages[getSelectedIndex(defaultLanguages, i18n)];
```
Comment (thisconnect) [inline frontends/web/src/components/terms/bitsurance-terms.tsx:37]: I think we should start using `i18n.resolvedLanguage ` like here https://github.com/digitalbitbox/bitbox-wallet-app/commit/bf70be385f422537b896ce6c853d425ee48f8f1e

**Review by thisconnect — CHANGES_REQUESTED** [review]

File: frontends/web/src/components/terms/bitsurance-terms.tsx (hunk 1 above)
Comment (Beerosagos) [reply frontends/web/src/components/terms/bitsurance-terms.tsx]: Nice! Updated also the one in bitsurance.tsx

**Comment by Beerosagos** [comment]
@thisconnect updated, PTAL :pray:

**Review by thisconnect — APPROVED** [review]
untested LGTM, please remove the console.log's :)

**Comment by Beerosagos** [comment]
> untested LGTM, please remove the console.log's :)

lol, oops

**Pull request merged**